usaspending_api/awards/tests/unit/test_location_filter_geocode.py

```diff
@@ -243,104 +237,3 @@ def test_get_fields_list():
-    assert (
-        to.filter(create_city_name_queryset("recipient_location", "transaction_id", ["Houston"], "USA", "TX")).count()
-        == 0
-    )
-
-
-def test_get_award_ids_by_city(award_data_fixture, elasticsearch_transaction_index):
-    elasticsearch_transaction_index.update_index()
-
-    # assert len(get_record_ids_by_city("nothing", "nothing", "nothing", "nothing", "nothing")) == 0
-
-    assert len(get_record_ids_by_city("pop", "award_id", "Houston", "USA", None)) == 1
-    assert len(get_record_ids_by_city("pop", "award_id", "Houston", "USA", "TX")) == 1
-    assert len(get_record_ids_by_city("pop", "award_id", "Houston", "USA", "VA")) == 0
-    assert len(get_record_ids_by_city("pop", "award_id", "Burbank", "USA", "CA")) == 0
-
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Burbank", "USA", None)) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Burbank", "USA", "CA")) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Burbank", "USA", "VA")) == 0
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Houston", "USA", "TX")) == 0
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "BRISTOL", "GBR")) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "BRISTOL", "FOREIGN")) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "BRISTOL", "USA")) == 0
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "BRISBANE", "FOREIGN")) == 0
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "BRISBANE", "USA")) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Burbank", "USA", None)) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Burbank", "USA", "CA")) == 1
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Burbank", "USA", "VA")) == 0
-    assert len(get_record_ids_by_city("recipient_location", "award_id", "Houston", "USA", "TX")) == 0
-
-
-def test_elasticsearch_results(award_data_fixture, elasticsearch_transaction_index):
-    elasticsearch_transaction_index.update_index()
-
-    query = {
-        "_source": ["award_id"],
-        "size": 0,
-        "query": {"match_all": {}},
-        "aggs": {"id_groups": {"terms": {"field": "award_id", "size": 5}}},
-    }
-    results = elasticsearch_results(query)
-    assert results is not None
-    assert len(results) == 5
-    assert results[0] == 1
-    assert results[1] == 2
-    assert results[2] == 3
-    assert results[3] == 4
-    assert results[4] == 5
-
-    query = {
-        "_source": ["award_id"],
-        "size": 0,
-        "query": {"match": {"pop_country": "XXXXX"}},
-        "aggs": {"id_groups": {"terms": {"field": "award_id", "size": 5}}},
-    }
-    results = elasticsearch_results(query)
-    assert results is not None
-    assert len(results) == 0
```

Comment:
Were these tests not relevant anymore?

Reply:
The functions being tested are removed in this PR